### PR TITLE
added file for subtraction

### PR DESCRIPTION
**Pull request by raghavmaheshwari92**
Please merge this to master.

### CODE REVIEW

File: sub.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def minus(a,b):
```
Comment (jatinkrmalik) [inline sub.py]: Where is the documentation?

File: sub.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+   c = a - b
+   print(c)
```
Comment (jatinkrmalik) [inline sub.py]: return c instead of printing.

File: sub.py (hunk 1 above)
Comment (raghavmaheshwari92) [reply sub.py]: Fixed. Please check.

File: sub.py (hunk 2 above)
Comment (raghavmaheshwari92) [reply sub.py]: Fixed. Please check.

**Pull request merged**